### PR TITLE
Add golangci-lint and fix lint errors / warning

### PR DESCRIPTION
**Pull request by moogle19**
I added golangci-lint to the travis file and fixes the errors / warnings  generated by golanci-lint.

### CODE REVIEW

File: hook.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ func (dummyFormatter) FormatQuery(b []byte, query string, params ...interface{})
 	return append(b, query...)
 }
 
+// QueryEvent ...
```
Comment (vmihailenco) [inline hook.go]: Hard to say if you got bored or missed this :) Anyway I guess we can address this later.

**Comment by vmihailenco** [comment]
Many thanks especially for the godoc comments.

File: orm/insert.go [hunk 2]
```diff
@@ -134,12 +134,13 @@ func (q *insertQuery) appendValues(b []byte, fields []*Field, strct reflect.Valu
 			continue
 		}
 
-		if q.placeholder {
+		switch {
+		case q.placeholder:
 			b = append(b, '?')
-		} else if (f.Default != "" || f.OmitZero()) && f.IsZeroValue(strct) {
+		case f.Default != "" || f.OmitZero() && f.IsZeroValue(strct):
```
Comment (betrok) [inline orm/insert.go]: It does not look like an equivalent replacement.

Comment (moogle19) [reply orm/insert.go]: Yeah, you are right. I forgot the brackets.

Comment (moogle19) [reply orm/insert.go]: I submitted a pull request ( #1193 ) fixing this issue. 
Thanks very much for finding this!

**Pull request merged**